Bugsnag/BSGOutOfMemoryWatchdog.m

```diff
@@ -274,17 +274,19 @@ - (NSDictionary *)readSentinelFile {
         bsg_log_err(@"Failed to read oom watchdog file: %@", error);
         return nil;
     }
-    NSDictionary *contents = [BSGJSONSerialization JSONObjectWithData:data options:0 error:&error];
+    NSMutableDictionary *contents = [[BSGJSONSerialization JSONObjectWithData:data options:0 error:&error] mutableCopy];
```

Comment:
In theory we could specify the `NSJSONReadingMutableContainers` option instead of manually making mutable copies...

Reply:
@kattrali beat me to it! 😅